Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -129,6 +129,13 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
           curl -LO \ |SCM_WEB|\/Documentation/installation/kind-config.yaml
           kind create cluster --config=kind-config.yaml
 
+.. note::
```

Comment:
I'm seeing this error in the docs build:

```
gettingstarted/k8s-install-default.rst:139: (INFO/1) No directive entry for "group-tab" in module "docutils.parsers.rst.languages.en".
gettingstarted/k8s-install-default.rst:13[9](https://github.com/cilium/cilium/actions/runs/6509963414/job/17682959036?pr=28522#step:4:10): (ERROR/3) Unknown directive type "group-tab".
gettingstarted/k8s-install-default.rst:155: (INFO/1) No directive entry for "group-tab" in module "docutils.parsers.rst.languages.en".
gettingstarted/k8s-install-default.rst:155: (ERROR/3) Unknown directive type "group-tab".
gettingstarted/k8s-install-default.rst:164: (INFO/1) No directive entry for "group-tab" in module "docutils.parsers.rst.languages.en".
gettingstarted/k8s-install-default.rst:164: (ERROR/3) Unknown directive type "group-tab".
```

I also noticed that the indentation for this will not put the note inside the kind tab. This might be related. I think that all of these lines need to be indented further so that the `..note::` lines up with the start of `.. parsed-literal::` above.